### PR TITLE
issue #14 to top button added

### PR DESCRIPTION
**Pull request by ronitblenz**
I have added the to top button which automatically scrolls to the top when clicked
{Smooth Animation Added}

Kindly merge my PR
(JWOC-MEDIUM)

Thanks

### CODE REVIEW

**Comment by ronitblenz** [comment]
Plz check my PR

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Don't attach the package-lock.json file in the code, please. Check the gitignore once if it's present there.

**Comment by ronitblenz** [comment]
I made the change you wanted, plz check and merge my PR

File: src/components/scroll/ScrollToTop.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { isVisible } from "@testing-library/user-event/dist/utils";
```
Comment (anushbhatia) [inline src/components/scroll/ScrollToTop.js:1]: add this library too in the package.json i think because of this deployment is failing

**Review by anushbhatia — CHANGES_REQUESTED** [review]
2 unused libraries are present because of which warnings are generated and your vercel deployment is failing
 `isVisible`, `BiWindows` kindly remove them if none of them are used

**Pull request closed without merging**